util/jsonutil/keywordsutil.go

```diff
@@ -0,0 +1,59 @@
+package jsonutil
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+type KeywordsString string
```

Comment:
Consider the type name `Keywords`.

Reply:
addressed in [98fe0a7](https://github.com/prebid/prebid-server/pull/2649/commits/98fe0a78fc69e18643709ac8c04b36e79154a776)